cli/src/main/resources/hudson/cli/client/Messages_ru.properties

```
@@ -0,0 +1,26 @@
+CLI.Usage=Jenkins CLI\n\
+  Использование: java -jar jenkins-cli.jar [-s URL] command [opts...] args...\n\
+  Параметры:\n\
+  \ -s URL              : URL сервера (по умолчанию используется переменная окружения JENKINS_URL)\n\
+  \ -webSocket          : Подключиться с помощью WebSocket (по умолчанию; хорошо работает с большинством обратных прокси-серверов; требует Jetty)\n\
+  \ -http               : Использовать пару соединений HTTP(S), а не WebSocket\n\
+  \ -ssh                : Использовать протокол SSH вместо WebSocket (требуется -user; порт SSH должен быть открыт на сервере)\n\
+  \ -i KEY              : Файл закрытого ключа SSH, используемый для аутентификации (для использования с -ssh)\n\
+  \ -noCertificateCheck : Полностью обойти проверку сертификата HTTPS. Используйте с осторожностью\n\
+  \ -noKeyAuth          : Не пытайтесь загрузить закрытый ключ аутентификации SSH. Конфликты с -i\n\
+  \ -user               : Указать пользователя (для использования с -ssh; необходимо зарегистрировать открытый ключ)\n\
+  \ -strictHostKey      : Запрос строгой проверки ключей хоста (для использования с -ssh)\n\
+  \ -logger FINE        : Включить подробное ведение журнала с клиента\n\
+  \ -auth [ USER:SECRET | @FILE ] : Указать имя пользователя и пароль или API-токен (или загрузить оба из файла);\n\
+  \                                 для использования с -http.\n\
+  \                                 Рекомендуется передавать учетные данные по файлам.\n\
+  \                                 Более подробную информацию и варианты смотрите на сайте https://www.jenkins.io/redirect/cli-http-connection-mode.\n\
+  \ -bearer [ TOKEN | @FILE ]     : Указать аутентификацию с использованием токена (или загрузить токен из файла);\n\
+  \                                 Для использования с -http. Взаимоисключающий вариант с -auth.\n\
+  \                                 Рекомендуется передавать учетные данные через файл.\n\
+  \n\
+  Доступные команды зависят от сервера. Выполните команду ''help'', чтобы \
+  посмотреть список.
+CLI.NoURL=Не указаны ни -s, ни переменная окружения JENKINS_URL.
+CLI.NoSuchFileExists=Не существует такого файла: {0}
+CLI.BadAuth=Параметры окружения JENKINS_USER_ID и JENKINS_API_TOKEN должны быть оба установлены или оставлены пустыми. 
```

Optional, would be nice to not have the trailing space character at the end of the line.
```suggestion
CLI.BadAuth=Параметры окружения JENKINS_USER_ID и JENKINS_API_TOKEN должны быть оба установлены или оставлены пустыми.
```